test/browsers.conf.js

```diff
@@ -4,30 +4,30 @@ module.exports = [
   {
     sessionName: 'Edge',
     name: 'Edge',
-    version: '86',
+    version: '100.0',
     os: 'Windows',
-    osVersion: '10',
+    osVersion: '11',
   },
   {
     sessionName: 'Firefox',
     name: 'Firefox',
-    version: '78.0',
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Use `91.13.0` for Firefox: it's the oldest version still supported (ESR) https://en.wikipedia.org/wiki/Firefox_version_history#Current_supported_official_releases